workbench-core/accounts/src/services/projectService.test.ts

```diff
@@ -2,50 +2,84 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
+const expectedUuid = '123';
+const projId = `proj-${expectedUuid}`;
+jest.mock('uuid', () => ({ v4: () => expectedUuid }));
 
 import {
   DynamoDBClient,
   GetItemCommand,
   GetItemCommandOutput,
   QueryCommand,
-  QueryCommandOutput
+  QueryCommandOutput,
+  UpdateItemCommand
 } from '@aws-sdk/client-dynamodb';
 import { marshall } from '@aws-sdk/util-dynamodb';
+import { AuthenticatedUser } from '@aws/workbench-core-authorization';
 import { mockClient } from 'aws-sdk-client-mock';
+import Project from '../models/project';
 import ProjectService from './projectService';
 
 describe('ProjectService', () => {
+  const ddbMock = mockClient(DynamoDBClient);
+  const TABLE_NAME = 'exampleDDBTable';
+  const projService = new ProjectService({ TABLE_NAME });
+  const timestamp = '2022-05-18T20:33:42.608Z';
+  const mockDateObject = new Date(timestamp);
+
   beforeAll(() => {
     process.env.AWS_REGION = 'us-east-1';
```

Comment:
Should we be storing the old `env` before we modify it? Something like what was implemented [here](https://github.com/aws-solutions/solution-spark-on-aws/blob/develop/workbench-core/accounts/src/services/accountService.test.ts#L14)

Reply:
to be honest, I do not know. It doesn't seem to be standardized for our unit tests. Maybe we should do a backlog ticket on that.